preload.js

```diff
@@ -154,14 +154,29 @@ const { getPlaceholderMigrations } =
 const { IdleDetector } = require('./js/modules/idle_detector');
 
 window.Signal = {};
+window.Signal.Backbone = require('./ts/backbone');
 window.Signal.Backup = require('./js/modules/backup');
 window.Signal.Crypto = require('./js/modules/crypto');
 window.Signal.Database = require('./js/modules/database');
 window.Signal.Debug = require('./js/modules/debug');
 window.Signal.HTML = require('./ts/html');
 window.Signal.Logs = require('./js/modules/logs');
 
-window.Signal.Components = {};
+// React components
+const { Lightbox } = require('./ts/components/Lightbox');
+const { MediaGallery } =
+  require('./ts/components/conversation/media-gallery/MediaGallery');
+
+const PropTypesMessage =
+  require('./ts/components/conversation/media-gallery/propTypes/Message');
```

Comment:
Why have Media-Gallery-specific `propTypes`? Wouldn't it make sense to make this a little wider, perhaps just `types`, pull it out to the top-level, right under `ts/components` or even `ts/`?